modules/ROOT/partials/configuration/menu.adoc

```diff
@@ -1,3 +1,4 @@
+[[menu]]
 == menu
```

Comment:
In case it's not clear, Mike is asking for backticks to be added around these, e.g.
```
`menu`
```
These were like this previously though, so do sort of fall out of the scope of this task. If you don't mind fixing them though it'd be appreciated.

Reply:
Done